client/api/src/main/java/org/opennms/velocloud/client/api/model/Edge.java

```diff
@@ -126,26 +124,26 @@ public static class Builder {
         private String logicalId;
         private String modelNumber;
         private String name;
-        private Boolean operatorAlertsEnabled;
+        private boolean operatorAlertsEnabled;
         private String selfMacAddress;
-        private Integer siteId;
+        private int siteId;
         private String softwareVersion;
 
         private Builder() {
         }
 
         public Builder withEnterpriseId(final UUID enterpriseId) {
-            this.enterpriseId = enterpriseId;
+            this.enterpriseId = Objects.requireNonNull(enterpriseId);
```

Comment:
This is not good style. If this setter is never called on the builder, the `enterpriseId` will still be `null`. While building, `null` is fine. Therefore the check should be moved back to the model.

Reply:
If we do not know about a default for the native types, using the object version in the builder is fine, too. But it should be native in the model.